ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__installer-rehearse-4.15.yaml

```diff
@@ -64,15 +64,16 @@ tests:
     env:
       BASE_DOMAIN: qe.azure.devcluster.openshift.com
     workflow: cucushift-installer-rehearse-azure-ipi-nat-gateway
+- as: installer-rehearse-ibmcloud-private
+  cron: '@yearly'
+  steps:
+    cluster_profile: ibmcloud-qe
+    workflow: cucushift-installer-rehearse-ibmcloud-ipi-private
```

Comment:
You can directly replace the install workflow in `installer-rehearse-ibmcloud` to run your testing, not have to add a new job.

Reply:
I want check the cucushift-installer-rehearse-ibmcloud-ipi-private and cucushift-installer-rehearse-ibmcloud-ipi both

Reply:
when you complete the testing, pls remove it.

Reply:
ok